Fix "TypeError: Cannot read property 'style' of null"

lovaiible opened the pull request:
### Checklist
- [x] I tested the changes and all related features on the test server https://artemistest.ase.in.tum.de.
- [x] I documented my source code using the JavaDoc / JSDoc style.
- [x] ~~I added integration test cases for the server (Spring) related to the features~~
- [x] ~~I added integration test cases for the client (Jest) related to the features~~
- [x] ~~I added screenshots/screencast of my UI changes~~
- [x] ~~I translated all the newly inserted strings~~

### Description
https://sentry.io/organizations/ls1intum/issues/1087612075/?project=1440029&query=is%3Aunresolved&statsPeriod=14d
Sentry logged an issue for the drag and drop click layer. This issue occurs when you navigate to a drag and drop quiz and navigate to another page right afterwards. Because of the time out, the click layer element is accessed when it is not rendered anymore and therefore leads to the null error.
![image](https://user-images.githubusercontent.com/33764166/61295283-fca52780-a7d7-11e9-906e-95a159220061.png)

I have reduced the time out duration to 100ms and added the check for the click layer element.

### Steps for Testing
1. Log in to ArTEMiS
2. Navigate to Course Administration
3. View dnd quiz in preview / edit / statistic view
4. Navigate to another page right afterwards before the dnd quiz has been completely rendered

### Screenshots
The screen doesn't change with this PR.

thilo-behnke commented on src/main/webapp/app/quiz-statistics/drag-and-drop-question-statistic/drag-and-drop-question-statistic.component.ts:
```diff
@@ -362,13 +362,18 @@ export class DragAndDropQuestionStatisticComponent implements OnInit, OnDestroy,
     }
 
+    /**
+     *  The background image for drag and drop quizzes is dynamically adjusted through scss, therefore we have to resize the click layer to the same width and height as the background image
```
Nice!

krusche commented:
Thanks for the improvement. However, I wonder why we have the exact same code 3 times? Could we not extract the code into a superclass or into a util class?

lovaiible commented:
Hi @krusche, I have changed that and extracted the function to a new utils file. The Travis build fails currently, but it should be fixed with this PR (https://github.com/ls1intum/Artemis/pull/655), doesn't it?
